documentation/FIWARE_COMPONENTS.md

```diff
@@ -0,0 +1,158 @@
+# Components and configuration
```

Comment:
@manuvaldi I think maybe it makese sense for this content to be appended to the UNDERSTANDING_FIWARE.md doc, instead of its own separated doc. What do you think?

Reply:
I would create all content and at the end reorg or fix to the better understanding.